src/clr-core/welcome.stories.ts

```diff
@@ -62,6 +62,12 @@ export const gettingStarted = () => {
 
       <p>
         Clarity Core is a Web Component implementation of the <a href="https://clarity.design">Clarity Design System</a>.
+        Clarity Core provides a set of reusable UI components that work in any JavaScript framework or no framework at all.
```

Comment:
I was hoping we'd see more content on the landing page, or just remove the current landing page and go straight to 'get started'

Reply:
Makes sense, I can update that

Reply:
Updated, added content to landing page as well as new browser support and improved framework docs. https://5e56e7bd7976455cb75d5a51--vmware-clarity.netlify.com/storybook/core